site/content/docs/5.0/examples/_index.md

```diff
@@ -14,7 +14,7 @@ aliases: "/examples/"
     {{- $len := len $entry.examples -}}
     {{ if (eq $i 0) }}<div class="row">{{ end }}
       <div class="col-sm-6 col-md-4 col-xl-3 mb-3">
-        <a href="/docs/{{ $.Site.Params.docs_version }}/examples/{{ $example.name | urlize }}/">
+        <a href="/docs/{{ $.Site.Params.docs_version }}/examples/{{ $example.name | urlize }}/"{{ if in $example.name "RTL" }} hreflang="ar"{{ end }}>
```

Comment:
I assume this is a leftover?

Reply:
No, this is needed since the link 's target isn't in the same language.